src/emailTemplates.js

```diff
@@ -36,7 +36,7 @@ export const baselineEmailTemplate = (data, isClinical) => {
     }
 
     return `
-        Dear ${data['399159511']},
+        Dear ${data['153211406'] !== '' ? data['153211406'] : data['399159511']},
```

Comment:
How about `data['153211406'] || data['399159511']`?

Reply:
this would execute `data['153211406']` if this is an empty string as well

Reply:
I think prefName is null on signup when it isn't filled in by the participant, so either of the following would cover that case in addition to the case you've covered:

```javascript
(data['153211406'] !== '' && data['153211406'] !== null && data['153211406'] !== undefined) ? data['153211406'] : data['399159511']

//or the far more succinct (this covers null, undef, and empty string)
data['153211406'] ? data['153211406'] : data['399159511']
```

Reply:
`A ? A : B` can be shortened to `A || B`.

Reply:
ill do ` data['153211406'] || data['399159511']`

Reply:
Good conversation guys!